BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/MetricTerms.cs

```diff
@@ -75,11 +76,23 @@ public static class PropertyNames
                 {
                     public const string Success = "Dependency.Success";
                     public const string TypeName = "Dependency.Type";
+                    public const string PerformanceBucket = "dependency/performanceBucket";
+                    public const string Target = "dependency/target";
+                }
+            }
+
+            public static class Common
+            {
+                public static class PropertyNames
+                {
+                    public const string CloudRoleInstance = "cloud/roleInstance";
+                    public const string CloudRoleName = "cloud/roleName";
+                    public const string Synthetic = "operation/synthetic";
                 }
 
-                public static class TypeNames
+                public static class PropertyValues
                 {
-                    public const string Other = "Other";
+                    public const string DimensionCapFallbackValue = "DIMENSION-CAPPED";
```

Comment:
Used to be DIMENSION_CAPPED, now DIMENSION-CAPPED. Is underscore to hyphen change intentional?

Reply:
Its insignificant. There are no backend dependency on this value.
Once we commit to this string, will inform backend teams on this - as of now only AppMap will rely on pre-aggs so it should be taught to be aware of dim caps.
For metric explorer/other metric experiences - Dimension=capped is just a string.